api/src/main/java/jakarta/enterprise/lang/model/AnnotationInfo.java

```diff
@@ -1,15 +1,29 @@
 package jakarta.enterprise.lang.model;
 
 import jakarta.enterprise.lang.model.declarations.ClassInfo;
+
+import java.lang.annotation.Annotation;
 import java.lang.annotation.Repeatable;
 import java.util.Collection;
 
-public interface AnnotationInfo {
+/**
+ * Models an annotation definition, providing access to the {@link AnnotationMemberValue}
+ * instances.
+ *
+ * @param <T>  The annotation type.
+ */
+public interface AnnotationInfo<T extends Annotation> {
+    /**
+     * The commonly used {@code value()} member.
+     */
+    String MEMBER_VALUE = "value";
+
     /**
      * Target of this annotation.
      * That is, the declaration, the type parameter or the type use on which this annotation is present.
      * TODO what if this annotation is a nested annotation?
      * TODO what if this annotation doesn't have a known target (e.g. qualifier of a synthetic bean)?
+     * TODO Do we need this? Retrieving the target from an annotation value is not supported in Micronaut
```

Comment:
I think we can probably remove this.

In Quarkus, the equivalent is used quite often, because we often start by simply looking up all annotations of some type, and then filter based on their target etc. But at the moment, I can't think of a way that would enable something like this in CDI Lite. I think we only allow top-down access, never bottom-up.

Reply:
Ok I will remove it in the PR